WordPress/Classes/Utility/ContextManager.swift

```diff
@@ -331,6 +330,26 @@ extension ContextManager {
     }
 }
 
+extension ContextManager {
+    /// - warning: This is designed to be used only for testing purposes.
+    func resetEverything() {
+        let container = persistentContainer.persistentStoreCoordinator
+        assert(container.persistentStores.count == 1)
+        guard let store = container.persistentStores.first, let storeURL = store.url else {
+            return assertionFailure()
+        }
+        do {
+            try container.destroyPersistentStore(at: storeURL, ofType: store.type, options: nil)
+            try FileManager.default.removeItem(at: storeURL)
+            persistentContainer.loadPersistentStores { _, error in
+                assert(error == nil)
+            }
+        } catch {
+            NSLog("failed to reset store: \(error)")
+        }
```

Comment:
To avoid this function being called accidentally, what do you think putting either the whole function declaration or just the function body inside `#if DEBUG`?

Reply:
I'm not sure you can accidentally call a function. I think it might be worth adding more of this infra under the conditional compilation. I wouldn't single out just this method. There are debug and test supporting infra throughout the app.